doc/articles/features/debugging-mobile.md

```diff
@@ -0,0 +1,44 @@
+---
+uid: Uno.Development.DebuggingOnMobile
+---
+
+# Debugging on Mobile
```

Comment:
This applies to Wasm as well? Basically, non-Desktop?

Reply:
Good point. We should update it, maybe connect it with the Wasm specific docs? "Debugging on Uno targets"

Reply:
This is for all mono targets. It looks more like a troubleshooting guide, let's call it Debugging troubleshooting.